spring-kafka-4/src/test/java/com/baeldung/saslplaintext/SprintContextTest.java

```diff
@@ -0,0 +1,15 @@
+package com.baeldung.saslplaintext;
+
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.test.context.junit.jupiter.SpringExtension;
+
+@ExtendWith(SpringExtension.class)
+@SpringBootTest(classes = KafkaSaslPlaintextApplication.class)
+class SprintContextTest {
+
+    @Test
+    void whenSpringContextIsBootstrapped_thenNoExceptions() {
+    }
+}
```

Comment:
Let's remove this since we already have KafkaSaslPlaintextApplicationLiveTest that would fail if there was any issue here